docs/source/release/v3.8.0/framework.rst

```diff
@@ -87,6 +89,12 @@ Improved
 - :ref:`ConvertUnits <algm-ConvertUnits>` now has the option to take a workspace with Points as input.
   A property has been added that will make the algorithm convert the workspace to Bins automatically. The output space will be converted back to Points.
 
+- :ref:`ConvertToHistogram <algm-ConvertToHistogram>`: Performance improvement using new HistogramData module,
```

Comment:
I am not sure how they got there, I must have made a mistake when merging. Fixed now.